repo_evaluate/csvcreator.py

```diff
@@ -13,7 +13,7 @@ def initialise(headers):
     :return: None
     :rtype: None
     """
-    with open('./results/result.csv', 'w', newline='') as f:
+    with open('./repo_evaluate/results/result.csv', 'w', newline='') as f:
```

Comment:
Results shouldn't go in the source directory. Maybe specify the file with a required argument?